src/metadata/modules.cpp

```diff
@@ -484,11 +484,29 @@ static HRESULT LoadSymbols(IMetaDataImport *pMD, ICorDebugModule *pModule, VOID
     IfFailRet(pModule->GetBaseAddress(&peAddress));
     IfFailRet(pModule->GetSize(&peSize));
 
+    std::vector<unsigned char> peBuf;
+    ULONG64 peBufAddress = 0;
+    if (isInMemory)
+    {
+        ToRelease<ICorDebugProcess> process;
```

Comment:
BTW, we probably don't need `process` before `if (peAddress != 0 && peSize != 0)` check, since in-memory module could be native image...

Reply:
Okay I made that change